sdk/python/kfmd/metadata.py

```diff
@@ -14,8 +14,8 @@
 
 import datetime
 import json
-import openapi_client
-from openapi_client import Configuration, ApiClient, MetadataServiceApi
+import kfmd.openapi_client as openapi_client
+from kfmd.openapi_client import Configuration, ApiClient, MetadataServiceApi
```

Comment:
style nit: prefer to import modules: 
http://google.github.io/styleguide/pyguide.html#22-imports

Reply:
Done.